adafruit_epd/epd.py

```diff
@@ -11,9 +11,17 @@
 
 import time
 from micropython import const
-from digitalio import Direction
+from digitalio import Direction, DigitalInOut
```

Comment:
I think the new `DigitalInOut import can be moved down into the try block, anywhere after the first `typing` import i.e.:

```
try:
    """Needed for type annotations"""
    from typing import Any, Union, Callable
    from digitalio import DigitalInOut
    from busio import SPI
    from PIL.Image import Image
except ImportError:
    pass
```